tlatools/org.lamport.tlatools/.classpath

```diff
@@ -32,5 +32,8 @@
 	<classpathentry kind="lib" path="lib/gson/gson-2.8.6.jar"/>
 	<classpathentry kind="lib" path="lib/lsp/org.eclipse.lsp4j.debug_0.21.1.v20230829-0012.jar"/>
 	<classpathentry kind="lib" path="lib/lsp/org.eclipse.lsp4j.jsonrpc_0.21.1.v20230829-0012.jar"/>
+	<classpathentry kind="lib" path="lib/junit-4.12.jar"/>
+	<classpathentry kind="lib" path="lib/easymock-3.3.1.jar"/>
```

Comment:
All three jars are part of Eclipse's target platform.  Does this cause conflicts, for example, when launching the Toolbox from Eclipse?

Reply:
I don't know, I can't get the toolbox to launch.